stage0/entrypoint.go

```diff
@@ -26,9 +26,9 @@ import (
 )
 
 const (
-	enterEntrypoint = "coreos.com/rkt/stage1/enter"
-	runEntrypoint   = "coreos.com/rkt/stage1/run"
-	gcEntrypoint    = "coreos.com/rkt/stage1/gc"
+	enterEntrypoint = "stage1-enter"
+	runEntrypoint   = "stage1-run"
+	gcEntrypoint    = "stage1-gc"
```

Comment:
The string "coreos.com/rkt/stage1/run" is not part of the appc spec. It is rkt-specific. So I would still namespace this with rkt:
```
runEntrypoint   = "rkt-stage1-run"
```

It is an ABI break between stage0 and stage1. So `rkt gc` will fail to garbage collect old pods from a previous version. Is it ok?